build_runner/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - __Breaking__: Remove support for hot-reloads. Use `package:webdev` instead.
 - Support version 2.0.x of the `build` package
 - Update to graphs `1.x`
+- Support version `1.x` of `shelf_web_socket` and `2.x` of `web_socket_channel`
```

Comment:
Should we do this in a non-breaking release? Maybe we should actually do this current release as non-breaking? It is only the hot reload support which was dropped which isn't really "breaking".

I don't want to do multiple breaking changes in quick succession here, and we will have another for null safety.

cc @natebosch wdyt?